packages/react-slider/src/components/Slider/useSliderState.tsx

```diff
@@ -69,7 +69,7 @@ const on = (element: Element, eventName: string, callback: (ev: any) => void) =>
 
 // The mark related classNames are needed since they are used in a JSX element that is dynamically generated.
 const markContainerClassName = 'ms-Slider-markItemContainer';
-const markClassName = 'ms-Slider-mark';
+export const markClassName = 'ms-Slider-mark';
```

Comment:
Thanks for doing this! The only thing you would need to check here is that the `index.ts` file does not export this from `useSliderState`, so instead of saying

```ts
export * from './useSliderState';
```

You do:

```ts
export { useSliderState } from 'useSliderState';
```